.vscode/settings.json

```diff
@@ -1,4 +1,6 @@
 {
   "editor.tabSize": 2,
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "prettier.printWidth": 7000,
+  "prettier.proseWrap": "never"
```

Comment:
This has caused unnecessary changes in the app(indentations print width and so ...on)
We already have it defined in the prettier config. you can remove this vscode settings folder json.